src/modules/commander/commander_params.c

```diff
@@ -924,6 +924,23 @@ PARAM_DEFINE_INT32(COM_ARM_CHK_ESCS, 0);
  */
 PARAM_DEFINE_INT32(COM_PREARM_MODE, 0);
 
+/**
+ * Enable pre-arm check on OpenDroneID.
+ *
+ * If this parameter is set, the pre-arm mechanism will check for the OpenDroneID system and prevent arming if it is not present.
+ *
+ * @group Commander
+ * @boolean
+ */
+PARAM_DEFINE_INT32(COM_PREARM_ODID, 0);
```

Comment:
>89.310(d) [standard]: The unmanned aircraft must be designed and produced in a way that reduces the ability of a person to tamper with the remote identification functionality.

Does a parameter satisfy this? Has anyone from the PX4/Ardupilot communities spoken with the FAA about this specifically? "reduces the ability" is kind of vague

Reply:
Sorry I just saw your above comment. Maybe we should add a comment like "Warning this parameter must be deleted and hardcoded to being enabled in a commercial product"

Reply:
> "reduces the ability" is kind of vague

It is. And to further muddy the waters, the FAA expanded a bit on that requirement in the [Notice of Availability](https://www.federalregister.gov/d/2022-16997/p-14) document. The ASTM remote ID working group is still discussing how exactly to respond to that.

The [MoC](https://github.com/opendroneid/opendroneid-core-c#united-states) has some limited text to address this requirement but it will in the end largely be up to the UA manufacturers to decide how they comply with this and then document their decision properly, in case the FAA at some point comes back and asks for details.